src/apps/main/database/collections/HashedSystemTreeCollection.ts

```diff
@@ -0,0 +1,121 @@
+import { DatabaseCollectionAdapter } from '../adapters/base';
+import { AppDataSource } from '../data-source';
+import { Repository } from 'typeorm';
+import * as Sentry from '@sentry/electron/main';
+import Logger from 'electron-log';
+import { ScannedItem } from '../entities/FileSystemHashed';
+
+export class HashedSystemTreeCollection implements DatabaseCollectionAdapter<ScannedItem> {
+  private repository: Repository<ScannedItem> = AppDataSource.getRepository('hashed_files');
```

Comment:
The naming could be in a `const` inside the entity so now you just have to change it from one place only.